bridge/bridgeapp/templates/registration/logged_out.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'home.html' %} {% block main %}
+<section>
+    <h2>You have successfully logged out. Click here to <a href="{% url 'login' %}">log in</a> again.</h2>
+</section>
+{% endblock %}}
```

Comment:
Have you ever seen a logged out page standing alone like this anywhere? What content you want to implement on the front end to go along with this?

You coded a login page and a separate register page--both prevent a user from accessing the app before being authenticated. Wouldn't a log-out take the user to the login page? If not where else the user would be taken to?

Please picture (or better yet, redraw the wireframe of) the app with the authentication templates that you attempt to implement and see if you're convinced they make sense, and reconsider and revise.